pagerduty/resource_pagerduty_user_contact_method.go

```diff
@@ -0,0 +1,173 @@
+package pagerduty
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+func resourcePagerDutyUserContactMethod() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePagerDutyUserContactMethodCreate,
+		Read:   resourcePagerDutyUserContactMethodRead,
+		Update: resourcePagerDutyUserContactMethodUpdate,
+		Delete: resourcePagerDutyUserContactMethodDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"user_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validateValueFunc([]string{
+					"email_contact_method",
+					"phone_contact_method",
+					"sms_contact_method",
+				}),
+			},
+
+			"send_short_email": {
+				Type:     schema.TypeString,
```

Comment:
This should probably be a `TypeBool`, don't you think? I would suggest that, and a default value for it, whatever is sensible for the resource

Reply:
Absolutely, good catch :+1: this should be solved now. Thank you so much! :)